digital-twin-model/src/sdv_v1.rs

```diff
@@ -10,19 +10,22 @@ fn context() -> Vec<String> {
     vec!["dtmi:dtdl:context;3".to_string(), "dtmi:sdv:context;1".to_string()]
 }
 
-/// Constants and type defintiions (including JSON serialization/deserialization) for each interface in the SDV model.
+/// Constants and type defintitions (including JSON serialization/deserialization) for each interface in the SDV model.
```

Comment:
```suggestion
/// Constants and type definitions (including JSON serialization/deserialization) for each interface in the SDV model.
```

(sorry, I think I provided an earlier correction that was also incorrect)

Reply:
Fixed.